cssgunc/CSPortal
# [Navbar] Added logo and made it mobile-friendly

## Summary 

Changed the color scheme of the navbar and added the RTC logo to it. Added a burger menu thingy to make it mobile-friendly.

## To test 

1. Check out the sample-pull-request branch `git checkout navbar`
2. Run `npm start` in your command line inside the RTCPortal root directory
3. Look at navbar

## Samples

**1. New color scheme:**

![New Color Scheme](https://user-images.githubusercontent.com/43911326/95666139-e7c35b00-0b24-11eb-8700-a4c5612b2b82.gif)

**2. Clickable Logo:**

![Logo Clickable](https://user-images.githubusercontent.com/43911326/95666151-fc075800-0b24-11eb-8d59-202227ba15f6.gif)

**3. Big to Small:**

![Big to Small](https://user-images.githubusercontent.com/43911326/95666160-0cb7ce00-0b25-11eb-841a-19a62bdb4d95.gif)

**4. Mobile menu:**

![Burger Menu on Mobile](https://user-images.githubusercontent.com/43911326/95666166-1b05ea00-0b25-11eb-879d-fe0d6fb585ea.gif)

**5. Mobile menu while navigating:**

![Closes when Navigating](https://user-images.githubusercontent.com/43911326/95666173-2a853300-0b25-11eb-9470-5c616a1208d0.gif)
